docs/content/docs/connectors/datastream/jdbc.md

```diff
@@ -167,15 +177,24 @@ env
                     ps.setDouble(4, t.price);
                     ps.setInt(5, t.qty);
                 },
-                JdbcExecutionOptions.builder().build(),
+                JdbcExecutionOptions.builder()
+                    .withMaxRetries(0)
+                    .build(),
                 JdbcExactlyOnceOptions.defaults(),
                 () -> {
                     // create a driver-specific XA DataSource
+                    // The following example is for derby 
                     EmbeddedXADataSource ds = new EmbeddedXADataSource();
                     ds.setDatabaseName("my_db");
                     return ds;
                 });
 env.execute();
 ```
+Postgres XADataSource Example:
+```java
+PGXADataSource pgxaDataSource = new PGXADataSource();
+pgxaDataSource.setUrl(
+"jdbc:postgresql://localhost:5432/postgres");
```

Comment:
I think users will likely need this too:
```
        xaDataSource.setUser(username);
        xaDataSource.setPassword(password);
```

Reply:
Yes, usually it should.

My intention for this example is to make this example "runnable out of box"; That says if user copy-paste the example to its IDE, and has postgres installed with default set up, it would work.

I installed postgre and `postgres` is the default table can be accessed without username and password or default (username and password).